README.md

```diff
@@ -62,21 +62,11 @@ operators][operators].
 
 ## Projects using operatorkit
 
-- https://github.com/giantswarm/app-operator
-- https://github.com/giantswarm/aws-operator
-- https://github.com/giantswarm/azure-operator
-- https://github.com/giantswarm/cert-operator
-- https://github.com/giantswarm/cluster-operator
-- https://github.com/giantswarm/chart-operator
-- https://github.com/giantswarm/flannel-operator
-- https://github.com/giantswarm/kvm-operator
-- https://github.com/giantswarm/node-operator
-- https://github.com/giantswarm/prometheus-config-controller
-- https://github.com/giantswarm/release-operator
-- more to come
+[Giant Swarm operators using operatorkit](https://github.com/search?p=1&q=topic%3Aoperator+org%3Agiantswarm&type=Repositories).
```

Comment:
Preparation for operator week. :trollface: 

I think it's easier to just tag the repos and we should do that anyway.